client/src/hooks/use-api.js

```diff
@@ -1,13 +1,13 @@
 import { useState, useEffect } from 'react';
 
-const API_ROOT = 'http://localhost:9000';
+const API_ROOT = 'https://czi-covid-lypkrzry4q-uc.a.run.app/api/exams';
```

Comment:
okay...
for sake of clarity, I believe its best if we don't change this for now as in I understand why this is here or why you changed this to adhere to what was originally in the cloned repo. As we are going to be performing requests (ie promise cycle) on various URLs we might just want to leave this as is, so revert it for now. Instead we will query this URL in the page we are rendering a response for as shown below

Reply:
will also add this to exam list issue